### PR TITLE
DB-2845: Support Inline Images In Drupal Body Field

### PR DESCRIPTION
**Pull request by mitchellmarkoff**
## What changes were made?
- Images embedded with media in a Drupal body field function OOTB in the `next-drupal-starter`

## Where were the changes made?
`next-drupal-starter`

## How have the changes been tested?
Images embedded with media in a Drupal body field have been observed to display properly in all templates.

## Additional information

Don't forget to [add a changeset](https://github.com/pantheon-systems/decoupled-kit-js#generating-a-changeset) if needed!

### CODE REVIEW

File: starters/next-drupal-starter/__tests__/__snapshots__/default/[...alias].test.jsx.snap [hunk 1]
```diff
@@ -235,43 +235,36 @@ exports[`default <CatchAllRoute /> > should render an article 1`] = `
           </h2>
           
 
-
```
Comment (CobyPear) [inline starters/next-drupal-starter/__tests__/__snapshots__/default/[...alias].test.jsx.snap:238]: 🤔 Seems kind of strange

File: starters/next-drupal-starter/__tests__/__snapshots__/umami/pages.test.jsx.snap [hunk 2]
```diff
@@ -260,6 +260,8 @@ exports[`umami <PageTemplate /> > should render with umami profile page 1`] = `
             <p>
               Umami is a fictional food magazine that has been created to demonstrate how you might build a Drupal site using functionality provided 'out of the box'.
             </p>
+
```
Comment (CobyPear) [inline starters/next-drupal-starter/__tests__/__snapshots__/umami/pages.test.jsx.snap:263]: Strange.

**Review by CobyPear — COMMENTED** [review]
I'm kind of confused about the snapshots. The code works fine but I wouldn't expect the snapshots to change like that.

File: starters/next-drupal-starter/__tests__/__snapshots__/umami/pages.test.jsx.snap (hunk 2 above)
Comment (mitchellmarkoff) [reply starters/next-drupal-starter/__tests__/__snapshots__/umami/pages.test.jsx.snap:263]: I also thought this was pretty strange but digging into it more it makes sense. In our example data, there are some instances where `body.value` seems to have more newline characters than `body.processed`.

Example:
 https://github.com/pantheon-systems/decoupled-kit-js/blob/cec912d94683e1ff56fdec7e57d8ff5ea49367ad/starters/next-drupal-starter/__tests__/data/topLevelArticleData.json#L6

https://github.com/pantheon-systems/decoupled-kit-js/blob/cec912d94683e1ff56fdec7e57d8ff5ea49367ad/starters/next-drupal-starter/__tests__/data/topLevelArticleData.json#L8

Comment (CobyPear) [reply starters/next-drupal-starter/__tests__/__snapshots__/umami/pages.test.jsx.snap:263]: Hmm ok good to know thanks for digging that out!

**Review by CobyPear — APPROVED** [review]

**Pull request merged**